app.py

```diff
@@ -20,6 +21,21 @@
 if __name__ == '__main__':
     commandLineArgs = command_line_args.getCommandLineArgs()
 
+    # Load database config
+    # notify user if the app will be using test or prod db
+    if (commandLineArgs.db == 'prod'):
+        print('Using production database...')
+        db_yaml = utils.get_yaml('data/database/config_prod.yml')
+    else:
+        print('Using local database...')
+        db_yaml = utils.get_yaml('data/database/config_dev.yml')
+        # delete local db from previous run
+        try:
+            # TODO: replace try...except with silent delete function
+            os.remove(db_yaml['database_credentials']['db_name'])
+        except OSError:
+            pass
+
```

Comment:
Should we go ahead and just create a general concept of environment for this? I think there is another story for that. Up to you.

Reply:
The implementation of the `silentremove` is in #26. I went ahead and added a place holder function for `silentremove` in `utils.py`, and called it in `app.py.` Maybe, just maybe, this will merge without conflict...